test/ansible-inventory/molecule/test-local/asserts.yml

```diff
@@ -5,7 +5,7 @@
     ansible_python_interpreter: '{{ ansible_playbook_python }}'
     cm: '{{ lookup("k8s", api_version="v1", kind="ConfigMap", namespace=namespace, resource_name="inventory-cm") }}'
   tasks:
-  - name: Assert sentinel ConfigMap has been created
+  - name: Assert sentinel ConfigMap has been created for Molecule Test
```

Comment:
I don't think this change is necessary, it's already under the molecule directory

Reply:
However, when we are trying to follow the logs is hard to know from where the log came and has more than one ansible test with these names.

Reply:
I am closing it as solved since after our chat it shows make sense for you too.